docs/notebooks/data_transformation.pct.py

```diff
@@ -244,6 +255,71 @@ def normalise(x, mean=None, std=None):
 print(f"observation: {observations[arg_min_idx, :]}")
 
 
+# %% [markdown]
+# # Data transformation with the DataTransformModelWrapper class
+#
+# Actually, we can achieve the same effect with a standard `BayesianOptimizer` by using a `DataTransformModelWrapper`.
```

Comment:
perhaps stress what is the difference/advantage? e.g. no need to manage the loop, several standard normalisers defined...

Reply:
a small separate paragraph below that explains in some details what happens below the hood?